apps/generator-cli/src/app/services/generator.service.ts

```diff
@@ -243,9 +243,17 @@ export class GeneratorService {
         )}" org.openapitools.codegen.OpenAPIGenerator`
       : `-jar "${cliPath}"`;
 
-    return [javaCmd, process.env['JAVA_OPTS'], subCmd, 'generate', appendix]
+    const command = [
+      javaCmd,
+      process.env['JAVA_OPTS'],
+      subCmd,
+      'generate',
+      appendix,
+    ]
       .filter(isString)
       .join(' ');
+
+    return this.isWin() ? `"${command}"` : command;
```

Comment:
what if `command` contains `"` ?

Reply:
i got it. i removed the `JAVA_HOME` env wrong. for now all tests are passed.

can you tell me how i can resolve the conflicts? why does this not behave like the first PR, where i was able to resolve the conflicts?

Reply:
you will need to cherry-pick 3e949d8839de87e985d7af4197eab54da2e1a8a3, which was reverted by me last week.

Reply:
> the generated commands already have " inside and only run successfully with these quotes in the windows build and without it will not.

did you test with something like `--additional-properties option="test-double-quote"` ?

Reply:
i tried to cherry-pick but after resolving the conflicts there was nothing to commit. i commited with `git commit --allow-empty` but it will not resolve the conflicts.


> did you test with something like --additional-properties option="test-double-quote" ?

i try to implement something like this to the test.

on my local env (windows - git bash) the following succeeded:

```sh
$ npm run oa generate -- -g ruby -i https://raw.githubusercontent.com/OpenAPITools/openapi-generator/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml -o ./out/ruby --additional-properties option="test-double-quote"

> examples@1.0.0 oa
> openapi-generator-cli generate -g ruby -i https://raw.githubusercontent.com/OpenAPITools/openapi-generator/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml -o ./out/ruby --additional-properties option=test-double-quote

[main] INFO  o.o.codegen.DefaultGenerator - Generating with dryRun=false
[main] INFO  o.o.codegen.DefaultGenerator - OpenAPI Generator: ruby (client)
[main] INFO  o.o.codegen.DefaultGenerator - Generator 'ruby' is considered stable.
...

$ echo $?
0
```